ann_benchmarks/datasets.py

```diff
@@ -192,6 +194,33 @@ def fashion_mnist(out_fn):
     test = _load_mnist_vectors('fashion-mnist-test.gz')
     write_output(train, test, out_fn, 'euclidean')
 
+# Creates a 'deep image descriptor' dataset using the 'deep10M.fvecs' sample
+# from http://sites.skoltech.ru/compvision/noimi/. The download logic is adapted
+# from the script https://github.com/arbabenko/GNOIMI/blob/master/downloadDeep1B.py.
+def deep_image(out_fn):
+    train_size = 10 * 1000 * 1000
+    test_size = 1000
```

Comment:
could we make the test set a bit bigger? 1k items make the recall numbers less reliable

Reply:
according to README.md this should be 10k also :)